KifuSF/Report.swift

```diff
@@ -21,18 +21,28 @@ enum FlaggedContentType: Int {
     case flaggedPhoneNumber = 100
     case flaggedCommunication
     
-    func getDescription() -> String {
+    var description: String {
```

Comment:
tell me what you think about this change?

Reply:
Computed property looks like the way to go for this case